README.md

```diff
@@ -81,46 +190,38 @@ or equivalently with an env var: `ORG_GRADLE_PROJECT_compileJavaJvmArgs='-Xmx2g
 By default, Lombok's annotation processor runs on the fly and Delomboked sources are not saved to {subproject}/build/delombok/...<br/>
 To enable Delombok, set `useDelombok` Gradle project property (e.g. `./gradlew -PuseDelombok build`).
 
-Local (non-CI) builds by default skip most of [Checker Framework's](https://checkerframework.org/manual/) checkers to speed up Java compilation.<br/>
-To make local builds more aligned with CI builds (at the expense of ~2x slower compilation from scratch),
-set `runAllCheckers` Gradle project property (e.g. `./gradlew -PrunAllCheckers build`).
-
 In case of spurious cache-related issues with Gradle build, try one of the following:
 * `./gradlew clean` and re-run the failing `./gradlew` command with `--no-build-cache`
 * remove .gradle/ directory in the project directory
 * remove ~/.gradle/caches/ (or even the entire ~/.gradle/) directory
 
 
-### Run & debug
-
-To run an instance of IDE with Git Machete IntelliJ Plugin installed from the current source,
-execute `:runIde` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `runIde` or `./gradlew runIde`).
-
-To watch the logs of this IntelliJ instance, run `tail -F build/idea-sandbox/system/log/idea.log`.
-
-
-### Run UI tests
-
-```
-./scripts/run-ui-tests [<intellij-version>]
-```
-
-See [Gradle Intellij plugin docs](https://github.com/JetBrains/gradle-intellij-plugin/tree/master/examples/ui-test-example)
-for more details.
-
-
 ### Generate plugin zip
 
 To generate a plugin archive run `:buildPlugin` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `buildPlugin` or `./gradlew buildPlugin`).<br/>
 The resulting file will be available under `build/distributions/`.
 
 
-### Install snapshot build of the plugin from CI
+## Issue reporting
```

Comment:
While you're here, pls also set up issue template (posssibly only the bug template is really needed) for this repository

Reply:
Template added, see PR #479